lib/beacon/loader.ex

```diff
@@ -3,6 +3,32 @@ defmodule Beacon.Loader do
 
   require Logger
 
+  defmodule Error do
```

Comment:
@TheFirstAvenger I'm trying to improve the error message when the module is not yet available. Wdyt? I'm accepting suggestions to make this message even better 🙌🏻